fcl/dunefd/gen/supernova/gkvm/prodmarley_nue_cc_gkvm_radiological_decay0_dune10kt_1x2x2.fcl

```diff
@@ -0,0 +1,10 @@
+# prodmarley_nue_cc_gkvm_radiological_decay0_dune10kt_1x2x2.fcl
+
+#include "prodmarley_nue_cc_flat_radiological_decay0_dune10kt_1x2x6_centralAPA.fcl"
+
```

Comment:
I am pointing out everywhere this appears so that if something is changed, it is easier to find